content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +34,180 @@ We have deprecated the following cert-manager APIs:
 - `acme.cert-manager.io/v1alpha3`
 - `acme.cert-manager.io/v1beta1`
 
-These APIs will be removed in cert-manager `v1.6.0`.
-If you have a cert-manager installation that is using or has previously used these deprecated APIs you may need to upgrade your cert-manager custom resources and CRDs. This needs to be done before upgrading to cert-manager `v1.6.0`.
-See cert-manager [docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources) for more detailed upgrade instructions.
+These APIs will be removed in cert-manager 1.6.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you have a cert-manager installation that is using or has previously used
+these deprecated APIs, you may need to upgrade your cert-manager custom
+resources and CRDs.
+
+This needs to be done before upgrading to cert-manager 1.6. See cert-manager
+[docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources)
+for more detailed upgrade instructions.
```

Comment:
suggestion: with this being in a warning which will draw peoples' attention, we should be really clear that this isn't an _immediate_ call to action.

```suggestion
⛔️  If you have a cert-manager installation that is using or has previously used
these deprecated APIs, you may need to upgrade your cert-manager custom
resources and CRDs before the release of cert-manager version 1.6.

See the ["remove deprecated APIs" docs](../../installation/upgrading/remove-deprecated-apis/#upgrading-existing-cert-manager-resources)
 for more detailed upgrade instructions.
```

Reply:
Richard suggested a new phrasing in 647c958 which removes the feeling of "it should be done immediately". Let me know if that's still feeling too much.